proto/kvrpcpb.proto

```diff
@@ -25,9 +25,9 @@ message KeyError {
 }
 
 enum CommandPri {
-    Normal = 0;
-    Low = -1;
-    High = 1;
+    Normal = 0; // Normal must the default value
```

Comment:
It just a number and we will not use the number but the name. The important point is that negative is inefficient.